packages/cli/src/api/catalog.ts

```diff
@@ -342,7 +342,7 @@ export function orderByMessage<T extends ExtractedCatalogType>(messages: T): T {
     .sort((a, b) => {
       const aMsg = messages[a].message || ""
       const bMsg = messages[b].message || ""
-      return aMsg.localeCompare(bMsg)
+      return aMsg.localeCompare(bMsg, "en-US")
```

Comment:
source language would be preferably always English in many projects, but still point is to have "stable" order in output

Reply:
Is there any drawbacks of hardcoding "en-US" as default collator locale? I'm trying to come up with something, but only thinks that could produce not-accurate sorting for some users which is fine comparing to unstable sort.

Reply:
yeah, we were investigating the code, and testing more versions, but this is the simplest as we don't have any information for example about "sourceLocale" in that function without bigger refactoring, but not it's really unstable and after upgrading to v4 we had a lot of complains about this issue :X

Reply:
yeah, we were investigating the code, and testing more versions, but this is the simplest as we don't have any information for example about "sourceLocale" in that function without bigger refactoring, but not it's really unstable and after upgrading to v4 we had a lot of complains about this issue :X

Reply:
I actually could make this "big" refactoring myself and pass a locale to the compare function from processed catalog, but the question - is it worth it?

Reply:
my 2c people will don't care about small sorting issues, if the order stay stable, so probably not worth it...